cypress/e2e/contactForm.cy.js

```diff
@@ -1,16 +1,16 @@
 import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 import HomeAndCataloguePageObject
-  from '../support/pages/homeСatalogue.pageObject';
-import faker from 'faker';
+  from '../support/pages/homeCatalogue.pageObject';
+import { faker } from '@faker-js/faker';
 /// <reference types='cypress' />
 
 const contactForm = new ContactFormPageObject();
 const homePage = new HomeAndCataloguePageObject();
 
 const testData = {
   email: faker.internet.email(),
-  name: faker.name.firstName(),
-  message: faker.random.words(),
+  name: faker.person.firstName(),
+  message: faker.word.words(),
```

Comment:
The use of `faker.person.firstName()` and `faker.word.words()` is correct for generating random data. Ensure that these methods are supported by the version of faker you are using.